apps/balancer-tools/src/components/Table.tsx

```diff
@@ -20,18 +17,6 @@ type TableColor = keyof typeof predefinedClasses;
 type TableVariant = keyof typeof predefinedClasses.blue;
 type TableShade = keyof typeof predefinedClasses.blue.solid;
 
-function useTableContext() {
-  const context = useContext(TableContext);
-
-  if (!context) {
-    throw new Error(
-      "Child components of Table cannot be rendered outside the Table component!",
-    );
-  }
-
-  return context;
-}
-
```

Comment:
🤔 curious about why you decided to remove this from here

Reply:
Our table use case on Historical APR needed a server component. Instead of creating a new table being copy and paste of this one but without `useTableContext` I decided to remove it from here.

But I will look at this again soon haahah I just had to look back at my PR because it has some functions that will be usefull here

Reply:
🤔 what if you added `use client` to the top of the file?

Reply:
did this and decided to not use suspense inside this table 👍